target/linux/realtek/rtl930x/target.mk

```diff
@@ -1,10 +1,11 @@
 # SPDX-License-Identifier: GPL-2.0-only
 ARCH:=mips
 SUBTARGET:=rtl930x
-CPU_TYPE:=24kc
```

Comment:
hmm, dropping this change, it builds and runs; but spewing illegal instruction.

rebuilding with 24kc to confirm this was causing it first. weirdly, the CPU IS being identified as 34kc ...

Reply:
The difference of `34kc` vs `24kc` is only relevant for the kernel. For userspace GCC generates identical code for both. So changing this from `24kc` to `34kc` has no advantage as the built binaries will still behave the same, but has the disadvantage of now requiring a `mips_34kc` stage 2 builder and its own package set.

No a NAK on this change.

Reply:
It's already gone, but I don't think this is true any longer. I'm getting 'Illegal instruction' errors with 34kc, and 24kc works fine. So technically this chip isn't truly 34kc compliant anyway it seems.

Anyway, what triggered me was the following patch: 308-mips32r2_tune.patch, which does some explicit 34kc stuff.

Reply:
The issue is/was more likely that there is no `CPU_CFLAGS_34kc` defined anymore in [include/target.mk](https://github.com/openwrt/openwrt/blob/master/include/target.mk#L212), so GCC probably defaulted to mips32r1. It was remove d in https://github.com/openwrt/openwrt/commit/a11843b62ebe75b552fa428556e3820657df3239.